frontend/ui/widget/touchmenu.lua

```diff
@@ -219,6 +219,12 @@ function TouchMenuBar:init()
             for i, sep in ipairs(self.icon_seps) do
                 local current_icon = i == k - 1 or i == k
                 self.icon_seps[i].style = current_icon and "solid" or "none"
+                if i == #self.icon_seps - 1 then
```

Comment:
If you like this change then I should add a comment explaining what this does first.